packages/cascara/src/ui/Pagination/Pagination.js

```diff
@@ -141,10 +143,10 @@ const Pagination = ({ totalRecordCount, state }) => {
               {!isLoading && (
                 <p className={styles.count}>{`of ${totalPages}`}</p>
               )}
-            </div>
-            <div className='field'>
-              <div className='ui small basic icon buttons'>
-                <button
+            </Role>
+            <Role className='field'>
+              <Role className='ui small basic icon buttons'>
+                <Button
```

Comment:
I just tested this on a screen reader, Buttons need a label as they are just read as 'Button', this is out of the scope of this PR, in the future we should revisit and add the proper labels to the buttons.

Reply:
Interesting, since this comes from reakit, I assume the accessibility should be fine. I'll make the change. Thank you Manu.

Reply:
@rociobaena no change needed, I was just making a note for a future task. 👍

Reply:
Agree, we could create a task to check all the proper labels not only to buttons.